Packages/PrimitivesComponents/Sources/ViewModels/TransactionViewModel.swift

```diff
@@ -71,7 +71,7 @@ public struct TransactionViewModel: Sendable {
                 Localized.Transfer.title
             }
         case .swap: Localized.Wallet.swap
-        case .tokenApproval: Localized.Transfer.Approve.title
+        case .tokenApproval: [Localized.Transfer.Approve.title, transaction.asset.symbol].joined(separator: " ")
```

Comment:
```suggestion
        case .tokenApproval: [Localized.Transfer.Approve.title, transaction.asset.symbol].joined(separator: " ")
```

keep as `Localized.Transfer.Approve.title` for consistency. 

Instead present symbol in the header